### ISSUE TITLE
Blur editor jumps to a nearby picture while dragging the image 

### ISSUE THREAD

**Issue by kaitu** [mapillary/mapillary_issues#2523]
### Basic information
* Platform: `Web`
* Version: `...` 
* System/Browser: `Firefox 52.02`

### Steps to reproduce behavior

 1. Load the blur editor
 2. Drag the image around

### Expected behavior
The image is not changed until some deliberate action (clear all, submit, move forward/backward) is taken.

### Actual behavior
The editor will suddenly skip to an image nearby (not necessarily belonging to the same sequence) 

### Additional information
I haven't found yet a way to reproduce at will. But the problem happens quite often with 360 images. Especially the skip to a different sequence is particularly annoying.

**Comment by katrinhumal** [comment]
@kaitu that sounds annoying! I've never seen this happen myself (which is totally unhelpful of course). Do you remember roughly when you started noticing this? Or how often you feel this happens? Since you say you can't reproduce then I guess providing the url of an image that this happened to won't help us much... but probably won't hurt either? Also, if you can see if you see any errors in the developer console of the browser?

**Comment by kaitu** [comment]
I think that I started to see the problem when I started shooting 360 images, which necessarely involve a lot of dragging. 

It's hard to tell how often it happens, I couldn't find a pattern. Since the editor is functional again I could review some 200 images without any occurrence, so far. But I try to do blind "clear all" request as often as I can by using memory or judgement, as I find dragging 360 images to check any part not only  excessively time wasting, but also (and worse) unbearably phisically straining. So the frequency depends a lot from the subject of the sequence.

Here is a photo from a sequence where jumpings to the adjacent sequence were particularly annoying (but probably also the only time when I shot on an already covered path):
https://www.mapillary.com/map/im/JSbtuPpyVqIIub1mNZjX7Q

Perhaps I should mention that to save time, I usually start dragging images as soon as I see something, before the transition from the previous image is complete and before blur markers are loaded.
